index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"
+    integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+  <script src="main.js"></script>
+  <title>Module 2-Project Reddit</title>
+</head>
+<body>
+  <div class="container">
+    <h1 class="module-2">ReReddit</h1>
+    <hr>
+    <div id="cont-posts"></div>
+    <form class="module-2" id="form-post" >
+      <div class="form-group">
+        <input type="text" class="form-control" id="text-from-post" placeholder="Post Text">
```

Comment:
the id needs to reflect the element, not the result of the value:
```suggestion
        <input type="text" class="form-control" id="post-text-input" placeholder="Post Text">
```